api/conversion/cluster.go

```diff
@@ -17,18 +17,32 @@ limitations under the License.
 package conversion
 
 import (
-	"github.com/pkg/errors"
+	"fmt"
 
+	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	cabpkv1a2 "sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/api/v1alpha2"
-	"sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/certs"
```

Comment:
:( all of the additions in this file are related to losing access to the certs helpers in newer versions of CABPK